src/rgw/rgw_rados.cc

```diff
@@ -1620,9 +1619,11 @@ void RGWPeriodMap::decode(bufferlist::iterator& bl) {
   }
 }
 
-int RGWPeriodMap::update(const RGWZoneGroup& zonegroup)
+int RGWPeriodMap::update(const RGWZoneGroup& zonegroup, CephContext *cct)
 {
   if (zonegroup.is_master && (!master_zonegroup.empty() && zonegroup.get_id() != master_zonegroup)) {
+    ldout(cct,0)<<"Error updating periodmap, multiple master zonegroups configured "<< dendl;
+    ldout(cct,0)<< "master zonegroup: " << master_zonegroup << " and  " << zonegroup.get_id() <<dendl;
```

Comment:
the formatting is still off here. can you add spaces between the `<<`s and arguments to `ldout()` please?